relayer/ops/monitoring/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build image
```

Comment:
just a general thought on the location of this file: would it better to be in the `chainlink-starknet/ops` folder rather than a new `relayer/ops` folder?

Reply:
done